sunpy/net/attr.py

```diff
@@ -57,32 +57,28 @@ def _print_attrs(attr, html=False):
     `str`
         String with the registered attributes.
     """
-    class_name = f"{attr.__module__+'.' or ''}{attr.__name__}"
     attrs = attr._attr_registry[attr]
-    sorted_attrs = make_tuple()
     # Only sort the attrs if any have been registered
-    if attrs.name:
-        sorted_attrs = _ATTR_TUPLE(*zip(*sorted(zip(*attrs))))
-    names = sorted_attrs.name
-    clients = sorted_attrs.client
-    names_long = sorted_attrs.name_long
-    descs = sorted_attrs.desc
-    descs = [x[:77] + '...' if len(x) > 80 else x for x in descs]
-    lines = []
-    t = Table(names=["Attribute Name", "Client", "Full Name",
-                     "Description"], dtype=["U80", "U80", "U80", "U80"])
-    for name, client, name_long, desc in zip(names, clients, names_long, descs):
-        t.add_row((name, client, name_long, desc))
-    lines.insert(0, class_name)
+    sorted_attrs = _ATTR_TUPLE(*zip(*sorted(zip(*attrs)))) if attrs.name else make_tuple()
+    *other_row_data, descs = sorted_attrs
+    descs = [dsc[:77] + ('...' if len(dsc) > 80 else '') for dsc in descs]
```

Comment:
```suggestion
    descs = [dsc[:77] + '...' if len(dsc) > 80 else dsc) for dsc in descs]
```
I think this will work.